advancing-bitcoin/2020/2020-02-06-andrew-chow-descriptor-wallets.es.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Andrew Chow - Carteras de descriptores  (2020-02-06)
+transcript_by: Michael Folkson
+translation_by: Blue Moon
+categories: ['taller']
+tags: ['cartera']
+---
+
+Nombre: Andrew Chow
+
+Tema: Repensar la arquitectura de las carteras: Carteras de descriptores nativos 
+
+Ubicación: El avance de Bitcoin
+
+Día: 6 de febrero de 2020 
+
+Vídeo: https://www.youtube.com/watch?v=xC25NzIjzog
```

Comment:
nit: valid markdown would be to put these in brackets like `https://www.youtube.com/watch?v=xC25NzIjzog`